recipes/siann/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "v1.0" %}
```

Comment:
Oh, one more thing. Please change the version to 1.0 and then for the url: use `v{{version}` instead of `{{version}}`. Version strings don't normally have letter prefixes and that can mess up sorting.